backend/pvm.pm

```diff
@@ -112,10 +112,12 @@ sub start_lpar {
     my $iso_present = qx/pvmctl media list -d VirtualOpticalMedia.media_name --where VirtualOpticalMedia.name=$iso/;
     if ($iso_present !~ /$iso/) {
         #copy over from nfs to VMLibrary
-        my $cmd = "sudo viosvrcmd --id 1 -r -c\" cp $source_iso /var/vio/VMLibrary/$iso\"";
-        runcmd($cmd);
-        $cmd = "sudo viosvrcmd --id 1 -r -c\" chown padmin:staff /var/vio/VMLibrary/$iso\"";
-        runcmd($cmd);
+        my @viocmd = ("sudo viosvrcmd --id 1 -r -c\" cp $source_iso /var/vio/VMLibrary/$iso\"");
```

Comment:
I remember again why I didn't merge it right away. I wondered why you put a space before the command here but not in the new call.